### PR TITLE
v6.5: update basic-features.md

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12232
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: basic-features.md [hunk 1]
```diff
@@ -10,192 +10,196 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [PITR](/br/backup-and-restore-overview.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
-[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
+[^2]: Starting from v6.5.0, the functions listed by querying the [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used for creating expression indexes in production environments, and more functions will be supported in future releases. For functions not listed by this variable, they are not recommended for use in a production environment. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```
Comment (qiancai) [inline basic-features.md:199]: @songrijie PTAL

Comment (songrijie) [reply basic-features.md:199]:
```suggestion
[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by system variable [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended in a production environment. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```

File: basic-features.md [hunk 2]
```diff
@@ -10,192 +10,196 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [PITR](/br/backup-and-restore-overview.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
-[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
+[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by system variable [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended in a production environment. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```
Comment (qiancai) [inline basic-features.md:199]:
```suggestion
[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in a production environment. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```

File: basic-features.md [hunk 3]
```diff
@@ -10,192 +10,196 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [PITR](/br/backup-and-restore-overview.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
-[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
+[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in a production environment. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```
Comment (qiancai) [inline basic-features.md:199]:
```suggestion
[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in production environments. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```

File: basic-features.md [hunk 4]
```diff
@@ -10,192 +10,196 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [Common Table Expressions (CTE)](/sql-statements/sql-statement-with.md)| Y | Y | Y | Y | Y | Y  | Y            |      Y       |      Y       |      Y       |      N       |      N       |
+| `START TRANSACTION`, `COMMIT`, `ROLLBACK`                    | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md)        | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) | Y | Y | Y | Y | Y | Y | Y            |      Y       |      Y       |      Y       |      Y       |      Y       |
+| [User-defined variables](/user-defined-variables.md)        | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [`BATCH [ON COLUMN] LIMIT INTEGER DELETE`](/sql-statements/sql-statement-batch.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:67]:
```suggestion
| [`BATCH [ON COLUMN] LIMIT INTEGER DELETE`](/sql-statements/sql-statement-batch.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
| [`BATCH [ON COLUMN] LIMIT INTEGER INSERT/UPDATE/REPLACE`](/sql-statements/sql-statement-batch.md) | Y | N | N | N | N | N | N | N | N | N | N | N |
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: basic-features.md [hunk 5]
```diff
@@ -10,192 +10,197 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [PITR](/br/backup-and-restore-overview.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
-[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
+[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [tidb_allow_function_for_expression_index](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in production environments. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```
Comment (TomShawn) [inline basic-features.md:200]:
```suggestion
[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [`tidb_allow_function_for_expression_index`](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in production environments. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```

**Review by shichun-0415 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

File: basic-features.md [hunk 6]
```diff
@@ -10,192 +10,197 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [`SHOW CONFIG`](/sql-statements/sql-statement-show-config.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`SET CONFIG`](/dynamic-config.md)                           | Y | Y | Y | Y | Y | Experimental| Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [DM WebUI](/dm/dm-webui-guide.md) | Experimental| Experimental| Experimental | Experimental| Experimental | Experimental | N | N | N | N | N | N |
+| [Foreground Quota Limiter](/tikv-configuration-file.md#foreground-quota-limiter)  | Y | Y | Y | Y| Experimental | Experimental | N | N | N | N | N | N |
+| [EBS volume snapshot backup and restore](https://docs.pingcap.com/tidb-in-kubernetes/v1.4/backup-to-aws-s3-by-snapshot)                        |    Y    |    Y    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
+| [PITR](/br/backup-and-restore-overview.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
```
Comment (TomShawn) [inline basic-features.md:196]:
```suggestion
| [PITR](/br/backup-and-restore-overview.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
| [Global memory control](/configure-memory-usage.md#configure-the-memory-usage-threshold-of-a-tidb-server-instance)   |    Y    |    Experimental    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
```

File: basic-features.md [hunk 7]
```diff
@@ -10,192 +10,198 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)                                 | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | N [^5]       |
+| [Database migration toolkit (DM)](/migration-overview.md)                                               | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | Y            |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                     | Y | Y | Y | Y | Y | Y  | Y   | Y    | Y    | Y    | Y    | Y    |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                   | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | Y            |
+| [Stream data to Amazon S3, Azure Blob Storage, and NFS through TiCDC](/ticdc/ticdc-sink-to-cloud-storage.md) | Experimental | N | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:172]:
```suggestion
| [TiCDC supports bidirectional replication across multiple clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | N | N | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md [hunk 8]
```diff
@@ -10,192 +10,199 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [Global memory control](/configure-memory-usage.md#configure-the-memory-usage-threshold-of-a-tidb-server-instance)   |    Y    |    Experimental    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 
 [^1]: TiDB incorrectly treats latin1 as a subset of utf8. See [TiDB #18955](https://github.com/pingcap/tidb/issues/18955) for more details.
 
-[^2]: See [Statement Reference](/sql-statements/sql-statement-select.md) for a full list of SQL statements supported.
+[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [`tidb_allow_function_for_expression_index`](/system-variables#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in production environments. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```
Comment (qiancai) [inline basic-features.md:202]:
```suggestion
[^2]: Starting from v6.5.0, the expression indexes created on the functions listed by the [`tidb_allow_function_for_expression_index`](/system-variables.md#tidb_allow_function_for_expression_index-new-in-v520) system variable have been tested and can be used in production environments, and more functions will be supported in the future releases. For functions not listed by this variable, the corresponding expression indexes are not recommended for use in production environments. See [expression indexes](/sql-statements/sql-statement-create-index.md#expression-index) for details.
```

File: basic-features.md [hunk 9]
```diff
@@ -10,192 +10,199 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)                                 | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | N [^5]       |
+| [Database migration toolkit (DM)](/migration-overview.md)                                               | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | Y            |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                     | Y | Y | Y | Y | Y | Y  | Y   | Y    | Y    | Y    | Y    | Y    |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                   | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | Y            |
+| [Stream data to Amazon S3, Azure Blob Storage, and NFS through TiCDC](/ticdc/ticdc-sink-to-cloud-storage.md) | Experimental | N | N | N | N | N | N | N | N | N | N | N |
+| [TiCDC supports bidirectional replication across multiple clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | N | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:172]:
```suggestion
| [TiCDC supports bidirectional replication between two clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | N | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 10]
```diff
@@ -10,192 +10,199 @@ This document lists the features supported in each TiDB version. Note that suppo
...
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)                                 | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | N [^5]       |
+| [Database migration toolkit (DM)](/migration-overview.md)                                               | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | Y            |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                     | Y | Y | Y | Y | Y | Y  | Y   | Y    | Y    | Y    | Y    | Y    |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                   | Y | Y | Y | Y | Y | Y  | Y           | Y            | Y            | Y            | Y            | Y            |
+| [Stream data to Amazon S3, Azure Blob Storage, and NFS through TiCDC](/ticdc/ticdc-sink-to-cloud-storage.md) | Experimental | N | N | N | N | N | N | N | N | N | N | N |
+| [TiCDC supports bidirectional replication between two clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | N | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:172]:
```suggestion
| [TiCDC supports bidirectional replication between two TiDB clusters](/ticdc/ticdc-bidirectional-replication.md) | Y | N | N | N | N | N | N | N | N | N | N | N |
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**